app/models/transaction.rb

```diff
@@ -87,6 +87,18 @@ def booking_uuid_object=(uuid)
     self.booking_uuid = UUIDUtils.raw(uuid)
   end
 
+  def community_uuid_object
+    if self[:community_uuid].nil?
+      nil
+    else
+      UUIDUtils.parse_raw(self[:community_uuid])
+    end
+  end
+
+  def community_uuid_object=(uuid)
```

Comment:
I think we could also live without the setter. The UUID shouldn't be changed after it's set the first time.